src/components/Select2.js

```diff
@@ -53,7 +69,7 @@ export default class Select2 extends Component {
   }
 
   render() {
-    const { data, ...params } = this.props;
+    const { data, value, ...params } = this.props;
```

Comment:
Problem with React warning was here, because select receive property `value`, but we not need it